group_labels.go

```diff
@@ -0,0 +1,191 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// GroupLabelsService handles communication with the label related methods of the
+// GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_labels.html
+type GroupLabelsService struct {
+	client *Client
+}
+
+// GroupLabel represents a GitLab group label.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_labels.html
+type GroupLabel Label
+
+func (l GroupLabel) String() string {
+	return Stringify(l)
+}
+
+// ListGroupLabelsOptions represents the available ListGroupLabels() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/labels.html#list-labels
+type ListGroupLabelsOptions ListOptions
+
+// ListGroupLabels gets all labels for given group.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_labels.html#list-group-labels
+func (s *GroupLabelsService) ListGroupLabels(gid interface{}, opt *ListGroupLabelsOptions, options ...OptionFunc) ([]*GroupLabel, *Response, error) {
```

Comment:
As this method is already on the `GroupLabelsService`, it's probably enough to call this `ListLabels`? I guess that goes for all method names, we can probably lose the `Group` part. What do you think?

Reply:
:+1: I agree. I just tried to be consistent with naming style in other `group_*.go` files. On it.

Reply:
Oh, I see... Well in that case then I prefer to keep things consistent. So let's not change it then!

Reply:
ok

Reply:
I see that you updated all the comments, so then I think we can just merge the PR now right? Anything else you wanted to add/change?

Reply:
I think it's good to go from my POV ... Maybe just one thing to consider is to add `Project` prefix to previous labels code since everything except labels uses prefixes to separate group and project resources.

Reply:
It would be breaking change tho...

Reply:
Agree, yet it does currently match how it names and documented in the Gitlab docs, so I think that's fine for now. Thanks again for your work!